docs/user-documentation/soc-and-feeds.md

```diff
@@ -0,0 +1,126 @@
+---
+title: SOC and Feeds
+id: soc-and-feeds
+slug: /soc-and-feeds
+sidebar_label: SOC and Feeds
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Swarm provides the ability to store content in content-addressed chunks or single owner chunks (SOC). With single owner chunks, a user can assign arbitrary data to an address and attest chunk integrity with their digital signature.
+
+Feeds are a unique feature of Swarm. They constitute the primary use case for single owner chunks. Feeds can be used for versioning revisions of a mutable resource, indexing sequential updates to a topic, publish the parts to streams, or post consecutive messages in a communication channel to name but a few. Feeds implement persisted pull-messaging and can also be interpreted as a pub-sub system.
+
+Because Feeds are built on top of SOCs, their interfaces in Bee is somewhat similar and they are using common concepts.
+
+## Single Owner Chunks
+
+The address of an SOC is calculated as the hash of an `identifier` and and `owner`. The `identifier` is a 32 bytes arbitrary data, usually expected as a hex string or in a `Uint8Array` type. The `owner` is the Ethereum address of the owner of the chunk, which is 20 bytes expected as a hex string or in a `Uint8Array` type.
+
+### Reading SOCs
+
+In order to read data from an SOC we need to make a reader object for a certain `owner`, then we can download the data with the provided `identity` that identifies the chunk.
+
+```js
+const owner = '0x8d3766440f0d7b949a5e32995d09619a7f86e632'
+const socReader = bee.makeSOCReader(owner)
+const identity = '0000000000000000000000000000000000000000000000000000000000000000'
+const soc = await socReader.download(identity)
```

Comment:
```suggestion
const identifier = '0000000000000000000000000000000000000000000000000000000000000000'
const soc = await socReader.download(identifier)
```
You used the name identifier earlier, imho it makes more sense

Reply:
It was a mistake, it's indeed called identifier